projects/tests.py

```diff
@@ -399,7 +399,7 @@ def test_invalid_project_permissions(self):
             "/api/projects/add-component/",
             {"creator": 0, "component_id": 1, "project_id": self.test_project.id},
         )
-        self.assertEqual(resp.status_code, 403)
+        self.assertEqual(resp.status_code, 404)
```

Comment:
This update is needed because before, users had 0 default permissions, but now they have some model-level permissions. The 403 was a result of model-level permissions checks being thrown, while the 404 is due to this user not having object-level permissions for the particular project being tested.